content/en/posts/css/introduction-to-cssnext/index.md

```diff
@@ -0,0 +1,368 @@
+}
+.h1:hover {
+  color: var(--secondary-Color);
+}
+body {
+  font-size: var(--r-Grid-baseFontSize, var(--r-Grid-baseFontSizeFallback));
+}
+```
+
+### Custom Media Queries
+
+Simply-named and semantic aliases ([check the specs](http://dev.w3.org/csswg/mediaqueries/#custom-mq)).
+
+```css
+@custom-media --viewport-medium (width <= 40rem);
+@custom-media --viewport-large (max-width: 50em);
+```
+
+Usage:
+
+```css
+@media (--viewport-medium) {
+  body { font-size: calc(var(--fontSize) * 1.2); }
+}
+@media (--viewport-large) {
+  body { font-size: calc(var(--fontSize) * 1.4); }
+}
+```
+
+### Custom selectors
+
+Take a look at the [specs](http://dev.w3.org/csswg/css-extensions/#custom-selectors) first.
+Let's say we want to apply some styles to all headings.
+
+```css
+@custom-selector :--heading h1, h2, h3, h4, h5, h6;
+
+:--heading {
+  margin-top: 0;
+}
+```
+This will output the following:
+
+```css
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 { margin-top; 0; }
+```
+
+### `color()`
+
+A simple [color function](http://dev.w3.org/csswg/css-color/#modifying-colors) to apply _color adjusters_ (hue, lightness, tint, and so on) to a base color.
+
+Examples:
+
+```css
+.class {
+  background-color: color(#2B88E6);
+  color: color(#2B88E6 red(+30) green(-50) blue(6%) alpha(.65));
+  border-top-color: color(#2B88E6 saturation(-8%) whiteness(+50%));
+  border-right-color: color(#2B88E6 lightness(5%) blackness(-25%));
+  border-bottom-color: color(#2B88E6 tint(80%));
+  border-left-color: color(#2B88E6 shade(75%));
+}
+```
+The code above will be transformed into...
+
+```css
+.class {
+  background-color: rgb(43, 136, 230);
+  color: rgba(73, 86, 15, 0.65);
+  border-top-color: rgb(181, 201, 222);
+  border-right-color: rgb(3, 45, 87);
+  border-bottom-color: rgb(213, 231, 250);
+  border-left-color: rgb(11, 34, 58);
+}
+```
+
+**cssnext** also offers the following color-related features.
+
+#### hwb()
+
+From the [specifications](http://dev.w3.org/csswg/css-color/#the-hwb-notation), HWB (Hue-Whiteness-Blackness) is similar to HSL but easier for humans to work with.
+
+```css
+.title {
+  color: hwb(125, 32%, 47%);
+}
+```
+
+Output:
+
+```css
+.title {
+  color: rgb(33, 135, 42);
+}
+```
+
+#### gray()
+
+Grays are [so cool](http://dev.w3.org/csswg/css-color/#grays) they have a function of their own.
+
+```css
+.section {
+  background-color: gray(120, 50%);
+  border-color: gray(17%, 25%);
+}
+```
+
+This will output:
+
+```css
+.section {
+  background-color: rgba(120, 120, 120, 0.5);
+  border-color: rgba(43, 43, 43, 0.25);
+}
+```
+
+#### #rrggbbaa
+
+**cssnext** transforms the [hexadecimal notations](http://dev.w3.org/csswg/css-color/#hex-notation) #RRGGBBAA and #RGBA into rgba().
+
+```css
+body {
+  color: #5c69;
+  background-color: #C73D5C59;
+}
+```
+
+Output:
+
+```css
+body {
+  color: rgba(85, 204, 102, 0.6);
+  background-color: rgba(199, 61, 92, 0.34902);
+}
+```
+
+#### rebeccapurple
+
+Simply transforms `rebeccapurple` into `rgb(102, 51, 153)`.
```

Comment:
Would this link do?
https://github.com/postcss/postcss-color-rebeccapurple#why-this-plugin-